sources/packages/api/src-sql/Notes/Create-institution-notes.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS sims.institution_notes (
+    institution_id INT NOT NULL REFERENCES sims.institutions(id),
```

Comment:
didn't we need `id SERIAL PRIMARY KEY,` in this table

Reply:
id added to institution_notes  and student_notes.